package.json

```diff
@@ -16,7 +16,7 @@
     "bluebird": "^3.4.7",
     "cheerio": "^0.22.0",
     "highlight.js": "^9.10.0",
-    "htmlparser2": "^3.9.2",
+    "htmlparser2": "https://github.com/MarkBind/htmlparser2/archive/v3.10.0-markbind.1.tar.gz",
```

Comment:
I believe if we push a new tag (like the v3.10 here) to the head of repo, we can do something like:
```
"htmlparser2": "MarkBind/htmlparser2#v3.10.0",
```

Reply:
Yep that works! The tag is `v3.10.0-markbind.1`, will be changing it.